src/Uno.UI/UI/Xaml/Controls/WebView/Native/Wasm/NativeWebView.wasm.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Web.WebView2.Core;
+using Uno.Foundation;
+using Uno.UI.Xaml.Controls;
+
+namespace Microsoft.UI.Xaml.Controls;
+
+public class NativeWebView : FrameworkElement, INativeWebView
+{
+	private CoreWebView2 _coreWebView;
+
+	public NativeWebView() : base("iframe")
+	{
+		this.HorizontalAlignment = HorizontalAlignment.Stretch;
+		this.VerticalAlignment = VerticalAlignment.Stretch;
+
+		this.RegisterEventHandler("load", OnNavigationCompleted, GenericEventHandlers.RaiseRoutedEventHandler);
+	}
+
+	private void OnNavigationCompleted(object sender, RoutedEventArgs e)
+	{
+		var uriString = this.GetAttribute("src");
+		Uri uri = null;
+		if (!string.IsNullOrEmpty(uriString))
+		{
+			uri = new Uri(uriString);
+		}
+		_coreWebView.RaiseNavigationCompleted(uri, true, 200, CoreWebView2WebErrorStatus.Unknown);
+	}
+
+	public void SetOwner(CoreWebView2 coreWebView)
+	{
+		_coreWebView = coreWebView;
+	}
+
+	public Task<string> ExecuteScriptAsync(string script, CancellationToken token)
+	{
+		var scriptString = WebAssemblyRuntime.EscapeJs(script);
+		return Task.FromResult(WebAssemblyRuntime.InvokeJS($"document.getElementById('{HtmlId}').contentWindow.eval(\"{scriptString}\")"));
```

Comment:
We should avoid using InvokeJS, it does not work in secure contexts. Let's make a support typescript method instead.